notes/stv/20200429-spark-tips-ML.txt

```diff
@@ -0,0 +1,170 @@
+#
+# <meta:header>
+#   <meta:licence>
+#     Copyright (c) 2020, ROE (http://www.roe.ac.uk/)
+#
+#     This information is free software: you can redistribute it and/or modify
+#     it under the terms of the GNU General Public License as published by
+#     the Free Software Foundation, either version 3 of the License, or
+#     (at your option) any later version.
+#
+#     This information is distributed in the hope that it will be useful,
+#     but WITHOUT ANY WARRANTY; without even the implied warranty of
+#     MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+#     GNU General Public License for more details.
+#  
+#     You should have received a copy of the GNU General Public License
+#     along with this program.  If not, see <http://www.gnu.org/licenses/>.
+#   </meta:licence>
+# </meta:header>
+#
+
+
```

Comment:
Is this a copy of this tutorial ?
https://www.datacamp.com/community/tutorials/apache-spark-tutorial-machine-learning

Reply:
Not a copy but that is the tutorial I followed (there's reference to that link in the notes as well) in the process of trying to learn a bit about ML and how to work with Dataframes and copied bits that I found useful and wanted to have . There is also a snippet from https://spark.apache.org/docs/latest/mllib-clustering.html